content/en/docs/tasks/access-application-cluster/configure-dns-cluster.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Configure DNS for a Cluster
+weight: 120
+---
+
+[Configure DNS for a Cluster](https://github.com/kubernetes/kubernetes/tree/release-1.5/examples/cluster-dns)
```

Comment:
@steveperry-53 @Bradamant3 @zacharysarah Did we decide our best practice for how to handle the adding a page because we can no longer add just a link in the nav side bar?  My preference would be to have a one or two sentence description before the link on this page just so it seems like a less choppy transition when the user shows up on this page.   Thoughts?  If the answer is we need to get this fixed ASAP and can add those later I'm fine with that. Just let me know.